pyproject.toml

```diff
@@ -6,12 +6,16 @@ authors = ["Meltano <hello@meltano.com>"]
 license = "Apache 2.0"
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<3.12"
+python = ">=3.7.1,<3.13"
 requests = "^2.31.0"
-singer-sdk = "^0.21.0"
+singer-sdk = "^0.32.0"
+greenlet = { version = ">=3", markers = "python_version >= '3.12'" }
+pendulum = { version = "3.0.0b1", markers = "python_version >= '3.12'" }
+pyyaml = { version = ">=6.0.1", markers = "python_version >= '3.12'" }
+typing-extensions = { version = ">=4.9.0", markers = "python_version >= '3.12'" }
 
```

Comment:
This one was also needed, but it should probably go into `dev-dependencies` instead.

Reply:
Dito.